net/wol/Makefile

```diff
@@ -1,6 +1,5 @@
 PLUGIN_NAME=		wol
-PLUGIN_VERSION=		1.0
-PLUGIN_REVISION=	1
+PLUGIN_VERSION=		2.0.d
 PLUGIN_DEPENDS=		wol
 PLUGIN_COMMENT=		Wake on LAN Service
 PLUGIN_MAINTAINER=	franco@opnsense.org
```

Comment:
but if I don't want it? ;)

Reply:
That's a different question - but not part of this PR ;)